week2-section1/exercise2/Main.cpp

```diff
@@ -55,14 +55,17 @@ class FMilitaryClock
 		{
 			std::cout << "0" << Hours;
 		}
-		
+		else if (Hours > 10)
+		{
+			std::cout << Hours;
+		}
 		if (Minutes < 10)
 		{
 			std::cout << "0" << Minutes;
 		}
-		else
+		else if (Minutes > 10)
```

Comment:
The bug is solved, but there are two comments here:
1. You've introduced a new bug. What if you input 10:10?
2. Remember that sometimes it makes sense to use an `else-if` but some othertimes the `else` is enough.